protocol/v2/ssv/validator/committee.go

```diff
@@ -293,6 +316,41 @@ func (c *Committee) ProcessMessage(logger *zap.Logger, msg *queue.SSVMessage) er
 	return nil
 
 }
+func (c *Committee) StopOldRunners(logger *zap.Logger, currentSlot phase0.Slot) error {
+	if maxRunnerAgeSlot > currentSlot {
+		return nil
+	}
+
+	firstTooOldSlot := currentSlot - maxRunnerAgeSlot
+
+	c.mtx.Lock()
+	defer c.mtx.Unlock()
+
+	for slot := range c.Runners {
+		if slot <= firstTooOldSlot {
```

Comment:
@MatheusFranco99  @MatusKysel 
do we have a message validation rule that would not allow messages from 32 slots ago?
i'm asking because it might be >32 (so 33 slots), in which case we'd need to replace the condition here from <= to <

Reply:
We have a rule for ["late messages"](https://github.com/ssvlabs/ssv/blob/stage/message/validation/common_checks.go#L37-L52). Committee messages are allowed up to 34 slots in the future and messages of other duty types are allowed up to 3.

Reply:
@moshe-blox I set the value `runnerExpirySlots` to 34 slots